src/DependencyInjection/Configuration.php

```diff
@@ -263,15 +257,13 @@ public function getConfigTreeBuilder()
                                                 ->arrayNode('roles')
                                                     ->prototype('scalar')
                                                         ->info('Roles which will see the route in the menu')
-                                                        ->defaultValue([])
                                                     ->end()
                                                 ->end()
                                                 ->arrayNode('route_params')
                                                     ->prototype('scalar')->end()
                                                 ->end()
                                                 ->booleanNode('route_absolute')
                                                     ->info('Whether the generated url should be absolute')
-                                                    ->defaultFalse()
```

Comment:
isnt this better than having to define it on the normalize method?

Reply:
The best to me was to use the default method for every node.

But we saw previously that if someone use `label: ~` the `null` value is not overridden by the default value.
Then we have two solutions:
- Remove `label: ~` in the documentation.
- Using the normalize method for the label.

But I feel weird to use the normalize method for some, and not the others.
Why could we write `label: ~` but not `roles: ~` ?

But I'm not fully satisfied too.

Reply:
Then maybe a solution is to maintain label: null but with a deprecation message (and convert it to empty string on the normalize method. And remove it from the docs, so people stop using it